### DISCUSSION TITLE
Docker image

### DISCUSSION THREAD

**Discussion by grzegorz-z1971** [kus/cs2-modded-server#25] [Ideas]
Hi! Is it possible to get a docker image for this modded server, so it can be easily self-hosted? 🙏

**Comment by tzachmost** [comment]
That would be mega-useful!

**Reply by DanielDKnudsen** [reply]
I'm somewhat new to creating Dockerfiles. But I played around with it today and got something working if you guys are still interested.

**Reply by tzachmost** [reply]
Yeah of course!

**Reply by DanielDKnudsen** [reply]
I made a fork of this repo here, works like a charm:

https://github.com/DanielDKnudsen/cs2-modded-server

**Reply by ghost** [reply]
I tried you docker version, but only get this on start:
Failed to initialize Steamworks SDK for gameserver.  Failed to load module '/home/steam/.steam/sdk64/steamclient.so'
